src/pages/signup/components/Signup2.tsx

```
@@ -142,18 +158,33 @@ const Signup2 = () => {
-            <Input
-              {...field}
-              inputtitle="생년월일"
-              placeholder="YYYY / MM / DD"
-              className={`h-[68px] w-full font-B02-M ${
-                errors.date ? 'border-warning' : ''
-              }`}
-              undertext={errors.date?.message}
-              undertextClassName={errors.date ? 'text-warning' : ''}
-            />
-          )}
+          render={({ field: { onChange, value, ...rest } }) => {
```

ESLint

[error] 'value' is defined but never used.

(@typescript-eslint/no-unused-vars)